### ISSUE TITLE
Generated documentation refers to python internal memory address when default argument value of a function is another function

### ISSUE THREAD

**Issue by pengparic01** [sphinx-doc/sphinx#1721]
Hi there,

In the following example, the argument add_seasonal_time of example_function() has a default value, which is the function add_seasonal_time() defined somewhere else.  When I build the documentation using Sphinx turns out it pointed to the internal memory address of the referred function.  Is this a bug that should be fixed?  Or did I miss something? Thanks a lot for your help!

```
example_function(decomp_df, 
                                reg_model_trend, 
                                cyc_arima_params=None, 
                                interval_minute=15, fcst_period=672, seasonal_days=7, conf_int=80,          
                                add_seasonal_time=<function add_seasonal_time at 0x108d821b8>)¶
```

**Comment by shimizukawa** [comment]
I think the problem is same as https://groups.google.com/d/msg/sphinx-users/0V77T-9cSSM/iHi8cNn7pQ0J

If it's true, it will be resolved by next 1.3 version.
The default behaviour, as of the latest development version, should be to just display `<function add_seasonal_time>`.  See #1694 for details.

**Comment by tk0miya** [comment]
Now this is already resolved.
Please reopen if you still have same problem with newest version.

Thanks,
